airflow/decorators/base.py

```diff
@@ -509,6 +509,9 @@ def _expand(self, expand_input: ExpandInput, *, strict: bool) -> XComArg:
             # task's expand() contribute to the op_kwargs operator argument, not
             # the operator arguments themselves, and should expand against it.
             expand_input_attr="op_kwargs_expand_input",
+            # start with trigger is not supported in dynamic task mapping
```

Comment:
Worth explaining why briefly in the comment.

Reply:
I thought it might not make sense for dynamic task mapping, but after a second thought, it might actually do. let me see how I can do that

Reply:
Yes, we're able to make it work with dynamic task mapping if `_start_trigger` and `_next_method` are defined as class attributes. However, if they're specified in `__init__`. It seems we do not actually run `__init__` before `schedulre_tis` but only validate kwargs https://github.com/apache/airflow/blob/90acbfbba1a3e6535b87376aeaf089805b7d3303/airflow/models/mappedoperator.py#L176. Is there anything I missed? If my understanding is correct, do you have suggestions on how this should be handled? Thanks!

Reply:
Just discussed with @uranusjr . We'll note this behavior in doc for now.